### ISSUE TITLE
Ability to view 'Completed' notes in default view.

### ISSUE THREAD

**Issue by geoffwellman** [inkdropapp/forum#47]
When I have marked a note as 'Completed' via the note status it will disappear from view when browsing notebooks. For me, this is not desirable behaviour because I always want to refer back to these notes.

It would be good if you could change the default notebook view to include 'Completed' based on a setting/preference somewhere as I always want to be able to browse my 'Completed' notes.

**Comment by craftzdog** [comment]
Hi, thank you for the suggestion.
The default filter is `-status:completed -status:dropped` which hides notes with a status of either completed or dropped, as you mentioned.
You can remove this filter from the search bar.
But doesn't it work for you?

**Comment by geoffwellman** [comment]
Yes, this doesn't work. I keep missing important notes I need to reference for new notes because of this filter. I think it would be useful for some to always show completed.

**Comment by craftzdog** [comment]
Let me discuss about it.
Adding new preference is easy but would make it complicated.
You will end up with searching from many options just like Adobe's products.

My first motivation of this feature is that I wanted to hide notes with 'Done' status.
Because I rarely refer past tasks.
I understand that's the personal preferences.

But how about tags?
It's useful to mark status of notes with color.
Please tell me what you think.

**Comment by geoffwellman** [comment]
Hi Craftzdog, happy to give an example of how I use it so you can understand.

One of my notebooks contains just meeting notes. I typically mark a meeting note as 'completed' post the meeting, once I have transferred any of my action items into my task management tool. A lot of these meetings are weekly, in following meetings I quite quickly need to refer to what was previously discussed, as the default view excludes 'completed' notes it means I cannot refer to previous notes as quickly as I'd like.

I could use tags but they are not quite as quick to use. 

You are right it is personal preference, I personally don't mind my completed or dropped notes always being in view because I typically leave my notes to be ordered by 'Last update date' and that means my most recent notes are in view.

**Comment by craftzdog** [comment]
Thank you for giving me your usecase!

I understood how you are using note status.
You have to break every meeting log down into tasks and you use note status to check which logs have been completed doing that.
But the meeting logs marked as completed are often necessary to refer back.
So this kind of notes is 'log' and also 'task', and you would like to treat as those.

I would think it's kind of special use-case.
I never intend to be negative with it though, it seems to be unexpected in design.
There should be alternative solutions other than adding preference.

My another suggestion is to use a sub-notebook for completed meeting logs like so:

![screen shot 2017-06-07 at 16 03 25](https://user-images.githubusercontent.com/1332805/26866147-ddcff092-4b9a-11e7-868e-f77a10607e71.png)

You can drag&drop notes into this notebook after transferring your actions into your todo app. It's quicker than tags.

I'd like to get someone else’s input on this issue.
Thanks!
